src/Fixer/FunctionNotation/VoidReturnFixer.php

```diff
@@ -203,8 +203,6 @@ private function hasVoidReturn(Tokens $tokens, $startIndex, $endIndex)
     }
 
     /**
-     * Apply the fix to the function definition.
```

Comment:
The same is true for many of the other methods in this class, which phpmd has not picked up though too.